app/src/main/java/com/imashnake/animite/features/theme/Theme.kt

```diff
@@ -12,16 +12,19 @@ import androidx.compose.runtime.Composable
 import androidx.compose.runtime.CompositionLocalProvider
 import androidx.compose.runtime.Immutable
 import androidx.compose.ui.platform.LocalContext
+import com.imashnake.animite.dev.ext.pastelize
 
 @Composable
 fun AnimiteTheme(content: @Composable () -> Unit) {
     val dynamicColor = Build.VERSION.SDK_INT >= Build.VERSION_CODES.S
     val darkTheme = isSystemInDarkTheme()
     val animiteColorScheme = when {
         dynamicColor && darkTheme -> dynamicDarkColorScheme(LocalContext.current)
-        dynamicColor && !darkTheme -> dynamicLightColorScheme(LocalContext.current)
+        dynamicColor && !darkTheme -> dynamicLightColorScheme(LocalContext.current).pastelize(
+            backgroundToPrimary = 0.05f
+        )
         darkTheme -> KimiNoDarkColorScheme
-        else -> KimiNoLightColorScheme
+        else -> KimiNoLightColorScheme.pastelize(backgroundToPrimary = 0.05f)
```

Comment:
Only 5% now?

Reply:
Yeah it gets too dark and it's hard to tell the cards from the background.